src/pages/packer/plugins/[...page].tsx

```diff
@@ -0,0 +1,168 @@
+import DocsView from 'views/docs-view'
+// Imports below are only used server-side
+import { cachedGetProductData } from 'lib/get-product-data'
+import {
+	generateStaticPaths,
+	generateStaticProps,
+} from 'components/_proxied-dot-io/packer/remote-plugin-docs/server'
+import { getPathBreadcrumbs } from 'components/breadcrumb-bar/utils/get-docs-breadcrumbs'
+import prepareNavDataForClient from 'layouts/sidebar-sidecar/utils/prepare-nav-data-for-client'
+import {
+	generateProductLandingSidebarNavData,
+	generateTopLevelSidebarNavData,
+} from 'components/sidebar/helpers'
+
+const basePath = 'plugins'
+const baseName = 'Plugins'
+/**
+ * Paths relative to the `website` directory of the Packer GitHub repo.
+ *
+ * Note that these are not currently used, as we don't yet support local
+ * preview for the dev dot UI. They've been retained to avoid too
+ * broad of a refactor to utilities shared with dot-io (where local
+ * preview is actively supported).
+ */
+const remotePluginsFile = 'data/plugins-manifest.json'
+const navDataFile = `data/${basePath}-nav-data.json`
+
+export async function getStaticPaths() {
+	let paths = await generateStaticPaths({
+		navDataFile,
+		remotePluginsFile,
+	})
+	paths = paths
+		// remove index-ish pages from static paths
+		.filter((p) => p.params.page.filter(Boolean).length > 0)
+		// limit number of paths to max_static_paths
+		.slice(0, __config.io_sites.max_static_paths ?? 0)
+	return {
+		paths,
+		fallback: 'blocking',
+	}
+}
+
+export async function getStaticProps({ params, ...ctx }) {
```

Comment:
This page re-uses utilities from the dot-io implementation, and merges and transforms the data returned from those utilities in order to meet the needs of our dev dot `DocsView` component.

I've tried to be thorough with comments here to ensure it's all clear. Intent here is to stub this implementation rather than have something perfect and polished, but very open to feedback 🙇‍♂️

Reply:
Very helpful comments here 👍